modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,143 @@ This release seeks to improve developer agility, reduce total cost of ownership,
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-652]
+== Release 6.5.2 (February 2021)
+
+Couchbase Server 6.5.2, released in February 2021, is the second maintenance release in the 6.5.x series for Couchbase Server. This release includes important bug fixes in various components.
+
+[#fixed-issues-652]
+=== Fixed Issues
+
+==== Analytics Service
+
+[#table_fixedissues_v652-analytics,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-44220[MB-44220^]
+| *Summary*: Fixed an issue where the Analytics driver went into a restart loop when a filesystem corruption was encountered, resulting in the Analytics service exiting with an error.
+
+| https://issues.couchbase.com/browse/MB-44215[MB-44215^]
+| *Summary*: The validity bit was used to determine corrupted merged components. 
+
+| https://issues.couchbase.com/browse/MB-44214[MB-44214^]
+| *Summary*: When making the debug log async, the DCP client log level was incorrectly set to ERROR. Fixed this issue to use the global service log level instead, which defaults to DEBUG.
+|===
+
+==== Cluster Manager
+
+[#table_fixedissues_v652-cluster-manager,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-42607[MB-42607^]
+| *Summary*: During TLS handshake with the Data Service, if the node certificate required more than 8192 bytes to transmit, then the handshake failed. This has been fixed.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v652-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-41868[MB-41868^]
+| *Summary*: Fixed an issue where histograms implemented with HDRHistogram stopped reporting when the total samples exceeded 2^31.
+
+| https://issues.couchbase.com/browse/MB-41867[MB-41867^]
+| *Summary*: Fixed an infinite loop due to HdrHistogram being reset.
+
+| https://issues.couchbase.com/browse/MB-41255[MB-41255^]
+| *Summary*: Fixed an issue where the replication of deletes with XATTRS failed if the affected document was non-resident.
+
+| https://issues.couchbase.com/browse/MB-40543[MB-40543^]
+| *Summary*: Added the ability to re-enable FTS logging in memcached for debug purposes.
+
+| https://issues.couchbase.com/browse/MB-40531[MB-40531^]
+| *Summary*: Fixed a performance regression where the HiFi-MRU Item Pager (added in 5.0) would evict more active items compared to previous LRU ItemPager under certain workloads.
+
+| https://issues.couchbase.com/browse/MB-38283[MB-38283^]
+| *Summary*: Fixed an issue where Memcached log wrapped due to excessive DCP "no memory" messages.
+|===
```

Comment:
Also need MB-41300 adding. I've asked Paolo to add a comment to that MB with details.